### DISCUSSION TITLE
YOURLS public stats but exclude some shorted urls for the public

### DISCUSSION THREAD

**Discussion by nudelsalat1** [YOURLS/YOURLS#3514] [Q&A]
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/YOURLS/.github/blob/main/CODE_OF_CONDUCT.md).

### Submission validity

- [X] This is not a personal support request, that should be posted on [discussions community](https://github.com/YOURLS/YOURLS/discussions).
- [X] I checked [current issues](https://github.com/YOURLS/YOURLS/issues) and this request isn't a duplicate of an existing issue, opened or closed.

### Summary

Hello,
I have unlocked the statistics (+) for the public.

`define('YOURLS_PRIVATE_INFOS', true);
define('YOURLS_NOSTATS', false); `

Is it possible that I exclude individual shortened URLs (sho.rt/EXAMPLE) for the public?
So that the statistics are only visible to me personally when logged in?
Is there a plugin for this, that I can exclude individual shorted url for the public in the interface?
Or do you have to specify that in the code?
I am grateful for any help!

### Motivation

Help for everyone

**Comment by ozh** [comment]
This would require a plugin. See https://yourls.org/docs/guide/extend/possible-with-a-plugin
